README.md

```diff
@@ -128,34 +148,40 @@ the `create-cloudfoundry` pipeline implements pipeline locking using
 This lock is acquired at the beginning and released the end of all the
 pipeline if it finishes successfully.
 
-In occasions it might be required to check the state or force the release
-of the lock. For that you can manually trigger the jobs `pipeline-check-lock`
+To check the state or force the release
+of the lock, you can manually trigger the jobs `pipeline-check-lock`
 and `pipeline-release-lock` in the job group `Operator`.
 
+## Optional flags
+### Override the branch used by pipelines
 
-## Optionally override the branch used by pipelines
-
-All of the pipeline scripts honour a
+All the pipeline scripts honour a
 `BRANCH` environment variable which allows you to override the git branch
 used within the pipeline. This is useful for development and code review:
 
 ```
 BRANCH=$(git rev-parse --abbrev-ref HEAD) make dev pipelines
 ```
 
-## Optionally override pipeline self updating
+Alternatively, you can use the `current-branch` option:
+```shell
+gds aws paas-dev-admin -- make dev02 current-branch pipelines
+```
 
-In case you want to prevent pipelines to self update, for example because you
-want to upload and test changes that you have made while developing, but not
-yet pushed to the branch pipeline is currently configured to pull from, you
-can use SELF_UPDATE_PIPELINE environment variable, set to false (true is default):
-`SELF_UPDATE_PIPELINE=false make dev pipelines`
+### Override pipeline self updating
+The pipelines are configured watch a specific branch and self-update when changes
+are pushed.
 
-## Optionally deploy to a different AWS account
+You can prevent this from happening by setting `SELF_UPDATE_PIPELINE=false` (it is true by default):
 
-See [doc/non_dev_deployments.md](doc/non_dev_deployments.md).
+```
+gds aws paas-dev-admin -- SELF_UPDATE_PIPELINE=false make dev pipelines
+```
+
+This can be useful when you want to upload and test changes that you have made while developing, but not
+yet pushed to the branch pipeline is currently configured to pull from
 
-## Optionally disable run of acceptance tests
+###  Disable run of acceptance tests
```

Comment:
If we're playing with this I might suggest changing this to "Disabling ...", otherwise being in the imperative it all reads like a list of instructions that must be performed.